src/components/Catalog/Catalog.scss

```diff
@@ -0,0 +1,24 @@
+@import '../../styles/utils/vars.scss';
+@import '../../styles/utils/mixins.scss';
+
+.catalogContent {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  margin: 0 auto;
+
+  max-width: 1168px;
+
+  @include onDesctop {
+    width: calc(3 * #{$productCard-width} + 2 * #{$productCards-gap-column});
+  }
+
+  @media (max-width: 920px) {
+    width: calc(2 * #{$productCard-width} + #{$productCards-gap-column});
+  }
```

Comment:
we don't have a 920px width point, and at 768px point there must be 3 cards, not two.